Applications/spire/source/book_view_ui_tester/random_book_view_model.cpp

```diff
@@ -93,6 +93,16 @@ QtPromise<void> RandomBookViewModel::load() {
         for(auto i = 0; i < 1000; ++i) {
           update();
         }
+        Quote bid_quote(Money(100), 100, Side::BID);
```

Comment:
It does not appear that Money(100) is significant or that this initial value is used in any way, as such I would recommend switching from Money(100) to just Money() as the former implies a significance, whereas the latter does not.